README.md

```diff
@@ -45,6 +45,8 @@ Option 1: Using [Composer](http://getcomposer.org/doc/00-intro.md) as dependency
 
     composer require lexpress/symfony1 "1.5.*"
     composer install
+    
+    Note: On windows, if your project is a few directories down from the drive root, composer can throw an error  relating to ZipArchive::extractTo(), this can be because pathnames are too long. There currently appears to be no proper solution but a workaround is to move your project to the drive root, run the commands from there, where they will run happily, and then move your project back. 
```

Comment:
Could you remove these 4 spaces to avoid your text being encapsuled in the code block just above ?